common/app/views/fragments/headerTopNav.scala.html

```diff
@@ -0,0 +1,206 @@
+@()(implicit page: model.Page, request: RequestHeader)
+
+@import common.{LinkTo, Edition}
+@import views.support.RenderClasses
+@import views.support.DropdownMenus.accountDropdownMenu
+@import navigation.AuthenticationComponentEvent._
+@import navigation.NavMenu
+@import navigation.UrlHelpers.{getJobUrl, Header, getReaderRevenueUrl}
+@import navigation.ReaderRevenueSite.{Support, SupportSubscribe, SupportContribute, PrintCTA, PrintCTAWeekly}
+@import conf.Configuration
+@import conf.switches.Switches.{IdentityProfileNavigationSwitch, SearchSwitch}
+
+@editionListItem(edition: Edition, isCurrentEdition: Boolean = false) = {
+<li class="dropdown-menu__item">
+    @defining(LinkTo(s"/preference/edition/${edition.id.toLowerCase}")) { href =>
+    <a data-link-name="nav3 : topbar : edition-picker: @edition.id" href="@href"
+        class="dropdown-menu__title @if(isCurrentEdition) {dropdown-menu__title--active}">
+
+        <span class="u-h">switch to the </span>
+        @edition.displayName
+    </a>
+    }
+</li>
+}
+
+@defining(NavMenu(page, Edition(request))) { navMenu: NavMenu =>
+<header class="@RenderClasses(Map(
+    " header-top-nav--slim" -> page.metadata.hasSlimHeader
+    ), s" pillar-scheme--${navMenu.currentPillar.map(_.title).getOrElse("")}", "header-top-nav"
+    )" role="banner">
+
+    <div class="header-top-nav__full-width">
+        <nav class="header-top-nav__inner gs-container" data-component="nav3" role="navigation"
+            aria-label="Guardian sections">
+
+            <div class="header-top-nav__top-bar">
+                @defining(Edition(request).id.toLowerCase()) { editionId =>
+                <div class="header-top-nav__item js-print-cta">
+                    <span class="header-top-nav__item--separator hide-until-desktop"></span>
+                    @defining(if (editionId == "uk") getReaderRevenueUrl(PrintCTA,
+                    Header) else getReaderRevenueUrl(PrintCTAWeekly, Header)) { href =>
+                    <a class="top-bar__item hide-until-desktop yellow" data-link-name="nav3 : print-cta"
+                        data-edition="@{editionId}" href="@href">
+                        @fragments.inlineSvg("newspaper", "icon", List("top-bar__item__icon"))
+                        Print subscriptions
+                    </a>
+                    }
+                </div>
+
+
+
+                <div class="header-top-nav__item">
+                    <span class="header-top-nav__item--separator hide-until-desktop"></span>
+                    <a class="top-bar__item js-navigation-sign-in my-account" data-link-name="nav3 : topbar : signin"
+                        href="@Configuration.id.url/signin?INTCMP=DOTCOM_NEWHEADER_SIGNIN&@createAuthenticationComponentEventParams(SigninHeaderId)">
+                        @fragments.inlineSvg("profile", "icon", List("top-bar__item__icon"))
```

Comment:
Should this inline svg have isPresentation = true so that it is given aria-hidden?

Reply:
Don't think you do need aria-hidden since they're already accompanied by text in the link!